packages/react-components/react-positioning/src/createPositionManager.ts

```diff
@@ -55,9 +55,13 @@ export function createPositionManager(options: PositionManagerOptions): Position
   const scrollParents: Set<HTMLElement> = new Set<HTMLElement>();
   const targetWindow = container.ownerDocument.defaultView;
 
-  // When the container is first resolved, set position `fixed` to avoid scroll jumps.
-  // Without this scroll jumps can occur when the element is rendered initially and receives focus
```

Comment:
- We don't need a focus hack as we will position an element before `.focus()` gets called
- This hack never worked for Floating UI as `position: fixed` gets removed right before `computePosition` (line 82)
- Initial styles are still needed as Floating UI checks for `position` value to determine `offsetParent`

Reply:
Okay, it's still need. I added a VR test to ensure that it works